postgresql/hooks/init

```diff
@@ -10,10 +10,12 @@ chown -R hab:hab {{pkg.svc_var_path}}
 
 if [[ ! -f "{{pkg.svc_data_path}}/PG_VERSION" ]]; then
   echo " Database does not exist, creating with 'initdb'"
-  initdb -U {{cfg.initdb_superuser_name}} \
-         -E {{cfg.initdb_encoding}} \
+  openssl rand -base64 32 > {{pkg.svc_config_path}}/pwfile
```

Comment:
Sometimes when you do this you can get this warning:

```
postgresql.default hook[init]:(HK): unable to write 'random state'
```

You can prevent this by doing:

```
export RANDFILE="{{pkg.svc_var_path}}/.rnd"
touch $RANDFILE
```

first.